src/ExportPDFPlugin.ts

```diff
@@ -38,15 +38,16 @@ export class ExportPDFPlugin extends Plugin {
   }
 
   initButtonCommands(): unknown {
-    return this.showButton
+    return ExportPDFPlugin.showButton
       ? {
-        '[picture_as_pdf] Export to PDF': EXPORT_PDF,
-      }
+          '[picture_as_pdf] Export to PDF': EXPORT_PDF,
+        }
       : {};
   }
 
   // this helps to invoke even in readonly mode.
-  perform(view: EditorView): boolean {
+  static perform(view: EditorView, showButton: boolean): boolean {
```

Comment:
Rename the static version to export, and leave the perform version in with a `@deprecated` jsdoc tag with a message to use the static export instead. Don't want this to be a breaking change yet.